docs/point_api_spec.md

```diff
@@ -0,0 +1,69 @@
+# Address Point API Spec
+
+## Purpose
+This document describes the Address Point Geocoding API, serving as a specification or "contract" for other users of this service.
+In particular, it describes the endpoints that this service exposes, as well as the structure and format of the data that is exchanged when using this service.
+
+### Protocols and data formats
+The Address Point Geocoding Service exposes data through HTTP, accepting and returing data in JSON format. 
+In particular, all requests need to specify their content type as appropriate:
+
+```
+Content-Type: application/json
+```
+
+Requests that don't specify their content type will be rejected
+
+### API Endpoints
+
+`GET  /status`
+
+Returns a message with current status and date
+
+```json
+{
+    "status": "OK",
+    "time": "Wed Feb 18 10:30:55 EST 2015"
+}
+```
+
+`POST /address/point`
```

Comment:
Also, I think there's a general [trend](https://blog.apigee.com/detail/restful_api_design_plural_nouns_and_concrete_names) [toward](http://www.restapitutorial.com/lessons/restfulresourcenaming.html) [plurals](https://github.com/WhiteHouse/api-standards#general-guidelines-for-restful-urls) in resources.  In this case, I guess that'd be:

`/addresses/points`

I prefer plural, but I think consistency is more important.  Whether we choose singular or plural, let's stick with it throughout our APIs.